
workaround this bug:

cc1: warnings being treated as errors
arch/mips/kernel/../../../fs/binfmt_elf.c: In function ‘vma_dump_size’:
arch/mips/kernel/../../../fs/binfmt_elf.c:1202: warning: ‘word’ may be used uninitialized in this function


--- a/arch/mips/kernel/Makefile	2009-02-16 11:25:13.000000000 +0000
+++ b/arch/mips/kernel/Makefile	2009-02-16 11:25:23.000000000 +0000
@@ -100,6 +100,5 @@
 
 obj-$(CONFIG_MIPS_CPUFREQ)	+= cpufreq/
 
-EXTRA_CFLAGS += -Werror
 
 CPPFLAGS_vmlinux.lds		:= $(KBUILD_CFLAGS)
